src/components/biggive-main-menu/biggive-main-menu.tsx

```diff
@@ -73,6 +73,10 @@ export class BiggiveMainMenu {
     });
     this.setHeaderSize();
 
+    // repeat setting header size in attempt to workaround issue in COM-102 comments with the total raised box
+    // being hidden behind menu in some cases on Safari.
+    window.setTimeout(() => this.setHeaderSize(), 200);
```

Comment:
did wonder if I need to check if we're actually running in a browser / if window is defined before calling window.setTimeout, but then if window was undefined we would have already crashed on line 71. Maybe that does happen or maybe stencil is setting a global window object anyway for SSR loads. Not sure what that would be or if it's better to change this to

```suggestion
    window && window.setTimeout(() => this.setHeaderSize(), 200);
```

Reply:
@bdsl I think [`globalThis`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis) is usually advisable whenever this is a question

Reply:
Right, the reason I've used window.setTimeout (or setInterval) rather than just plain function calls is that the return type is different between browser and node. Doesn't matter here since I'm not using the return. There's no real reason to change it here as we already have a call to a window function a few lines up is there though @NoelLH ?

Reply:
No, you're right @bdsl I can't see a way one would be worse than the other. Even if we wanted to try changing it, makes sense to do one thing at a time and assess at each step.